curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md

```diff
@@ -3,26 +3,24 @@ id: 587d7fb1367417b2b2512bf4
 title: Chain Middleware to Create a Time Server
 challengeType: 2
 forumTopicId: 301510
-localeTitle: Промежуточное ПО для создания сервера времени
+localeTitle: Middleware для создания сервера времени
 ---
 
-## Description
+## Описание
```

Comment:
@RandellDawson is it ok - translating such this line?

Reply:
Nope. I think it should remain `Description` itself.